tests/src/Kernel/Entity/GetEntityGroupsTest.php

```diff
@@ -0,0 +1,216 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\Tests\og\Kernel\Entity\GetEntityGroupsTest.
+ */
+
+namespace Drupal\Tests\og\Kernel\Entity;
+
+use Drupal\entity_test\Entity\EntityTest;
+use Drupal\KernelTests\KernelTestBase;
+use Drupal\Component\Utility\Unicode;
+use Drupal\og\Entity\OgMembership;
+use Drupal\og\Og;
+use Drupal\user\Entity\User;
+
+/**
+ * Tests getting the memberships of an entity.
+ *
+ * @group og
+ */
+class GetEntityGroupsTest extends KernelTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = ['system', 'user', 'field', 'og', 'entity_test'];
+
+  /**
+   * @var \Drupal\user\Entity\User
+   */
+  protected $user1;
+
+  /**
+   * @var \Drupal\user\Entity\User
+   */
+  protected $user2;
+
+  /**
+   * @var \Drupal\user\Entity\User
+   */
+  protected $user3;
+
+  /**
+   * @var \Drupal\entity_test\Entity\EntityTest
+   */
+  protected $group1;
+
+  /**
+   * @var \Drupal\entity_test\Entity\EntityTest
+   */
+  protected $group2;
+
+  /**
+   * The machine name of the group node type.
+   *
+   * @var string
+   */
+  protected $groupBundle;
+
+  /**
+   * The machine name of the group content node type.
+   *
+   * @var string
+   */
+  protected $groupContentBundle;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+
+    $this->installConfig(['og']);
+    $this->installEntitySchema('og_membership');
+    $this->installEntitySchema('user');
+    $this->installEntitySchema('entity_test');
+    $this->installSchema('system', 'sequences');
+
+    $this->groupBundle = Unicode::strtolower($this->randomMachineName());
+    $this->groupContentBundle = Unicode::strtolower($this->randomMachineName());
+
+    // Create users.
+    $this->user1 = User::create(['name' => $this->randomString()]);
```

Comment:
Indeed. This was copied from another tests. Replacing all of them would be
a good idea.
On 7 Dec 2015 17:54, "Daniel Wehner" <notifications@github.com> wrote:

> In tests/src/Kernel/Entity/GetEntityGroupsTest.php
> <https://github.com/amitaibu/og/pull/53#discussion_r46852535>:
>
> > +   * {@inheritdoc}
> > +   */
> > +  protected function setUp() {
> > +    parent::setUp();
> > +
> > +    $this->installConfig(['og']);
> > +    $this->installEntitySchema('og_membership');
> > +    $this->installEntitySchema('user');
> > +    $this->installEntitySchema('entity_test');
> > +    $this->installSchema('system', 'sequences');
> > +
> > +    $this->groupBundle = Unicode::strtolower($this->randomMachineName());
> > +    $this->groupContentBundle = Unicode::strtolower($this->randomMachineName());
> > +
> > +    // Create users.
> > +    $this->user1 = User::create(['name' => $this->randomString()]);
>
> random strings make it harder to debug, I'd just go with hardcoded strings
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/amitaibu/og/pull/53/files#r46852535>.
>